Mathlib/Topology/Order/Category/FrameAdjunction.lean

```diff
@@ -0,0 +1,120 @@
+/-
+Copyright (c) 2023 Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen, Sam v. Gool, Leo Mayer, Brendan S. Murphy
+-/
+import Mathlib.Topology.Category.Locale
+
+/-!
+# Adjunction between Locales and Topological Spaces
+
+This file defines functors between the categories of Locales and Topological Spaces
```

Comment:
```suggestion
This file defines functors between the categories of Locales and of Topological Spaces
```

Reply:
I will turn this into lowercase (and add the of). Thanks